### ISSUE TITLE
Ability to delete saved queries (Named operations) in the UI

### ISSUE THREAD

**Issue by p013570** [gchq/gaffer-tools#792]
Once a user has saved a query in the UI they need a way to delete them in the UI.

**Comment by n3101** [comment]
The UI fails to meet our Accessibility policy. It is unmaintainable and will be removed shortly.
